charts/k8ssandra/values.yaml

```diff
@@ -80,13 +80,26 @@ cassandra:
     #size: 800M
     #newGenSize: 400M
 
-
 stargate:
   # configuration regarding the Stargate API
   enabled: false
   replicas: 1
   clusterVersion: "3.11" # this is the only supported value for now
   containerImage: "auto" # chooses image based on clusterVersion
+  cassandraUser:
+    secret: ""
+    username: ""
+
+# cassandraUser configures the Cassandra user used by Stargate when authentication is
```

Comment:
Minor nit - can you move this comment block above line 89 and indent the block 2 more spaces?